### ISSUE TITLE
Latest version 0.39.0 fails to compile & build successfully

### ISSUE THREAD

**Issue by ravindran-a** [ivpusic/react-native-image-crop-picker#1878]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.39.0
- react-native v0.70.6

### Platform
Tell us to which platform this issue is related
- Android

### Expected behaviour
Expected to build successfully

### Actual behaviour
Build failing

### Attachments

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/19164536/207653713-fab12f0c-9aa3-4948-bed8-612195a4d565.png">

**Comment by EslamElMeniawy** [comment]
ReactNative v0.70.6 uses `compileSdkVersion=31` so `Build.VERSION_CODES.TIRAMISU` is not available as it is available from version 33 only.
So for workaround now I changed the compile SDK version in my project to be 33 to be able to build and run the app.

**Comment by acarkaan** [comment]
> ReactNative v0.70.6 uses `compileSdkVersion=31` so `Build.VERSION_CODES.TIRAMISU` is not available as it is available from version 33 only.
> So for workaround now I changed the compile SDK version in my project to be 33 to be able to build and run the app.

solved the problem. thank u.

**Comment by nschild** [comment]
Please mark 0.39.0 as a breaking change. CompilesSdkVersion >= 31 requires a minimum of react native 0.65 or 0.64 (I don't remember which) and a Gradle update as well. And, as in our situation, almost every native library has to be updated as this now requires the exported attribute on all the various tags in the android manifests. 

Could be an addendum to the existing note or as other libraries do it in a table format.
"If you are using react-native >= 0.60 use react-native-image-crop-picker version >= 0.25.0. Otherwise use version < 0.25.0."

**Comment by AntonioLi1** [comment]
> ReactNative v0.70.6 uses `compileSdkVersion=31` so `Build.VERSION_CODES.TIRAMISU` is not available as it is available from version 33 only. So for workaround now I changed the compile SDK version in my project to be 33 to be able to build and run the app.

Thanks for the solution!
Is this a short term fix or will it be for the long term?

**Comment by log901108** [comment]
any further fix here?

**Comment by cyhnkckali** [comment]
Will there be any fixes for this issue?

**Comment by priyanu17** [comment]
Facing Same problem , does not compile on android, ios is working

**Comment by Reagankm** [comment]
Yeah. Compile version 33 is not workable for me.  That only runs on 7% of Android devices. Whereas version 27, like it states in the README, covers 90%

**Comment by RodolfoGS** [comment]
Same issue, in the meantime I'll stay on 0.38.1, since the changes of 0.39.0 are not too relevant for me.

https://github.com/ivpusic/react-native-image-crop-picker/releases/tag/v0.39.0

**Comment by tapz** [comment]
Setting compileSdkVersion=33 in build.gradle ofr my app does not help. Still seeing the same error.

**Comment by luchozamora** [comment]
As @RodolfoGS said, staying on version 0.38.1 should be enough.

Simply downgrade with
`yarn add react-native-image-crop-picker@0.38.1`

**Comment by CodingWithNobody** [comment]
Check this https://www.youtube.com/watch?v=Bx_gwRODtlY
